configure.ac

```diff
@@ -134,6 +145,13 @@ AC_ARG_ENABLE([data-download],
                 *) AC_MSG_ERROR([bad value ${enableval} for --disable-data-download]) ;;
               esac], [DOWNLOAD_DATA=true])
 
+AC_ARG_ENABLE([senzing-datamodel],
```

Comment:
Can we make it “—model=senzing” and then if there are other models down the road that flag can be a switch?

Reply:
I like your suggestion. However, I couldn't find a way to use `—model=` so I went with environment variable: `MODEL=senzing` 
```
./configure --datadir=[path to datadir] MODEL=senzing
```
I'm no autoconfig expert so if you know of a way to use `—model=` then let me know.

Reply:
hm yeah have mostly been in Python CLI land recently, guess there's not really a way to add an arbitrary command-line arg with autoconf, but MODEL= does the trick